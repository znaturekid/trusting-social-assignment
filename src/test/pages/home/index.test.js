import React from 'react';
import Enzyme, {mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import HomeComponent from '../../../pages/home';
import { mountWrap, shallowWrap } from '../../help';
import configureMockStore from 'redux-mock-store';

import thunk from 'redux-thunk';
import { Provider } from 'react-redux';;

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

Enzyme.configure({
    adapter: new Adapter()
});

const store = mockStore({
    news: {
        newLists: [],
        page: 1,
        query: '',
        loading: false
    }
})
const props = {
    
}
const component = (<Provider store={store}><HomeComponent></HomeComponent></Provider>)

describe('components', () => {
    
    describe('Home =>', () => {
        it('should render home correctly', () => {
            const home = shallowWrap(component);
            expect(home).toMatchSnapshot();
        })
    })
})