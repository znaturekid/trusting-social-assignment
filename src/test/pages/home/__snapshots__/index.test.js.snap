// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Home => should render home correctly 1`] = `<HomePage />`;
